courses/free-courses-ml.md

```diff
@@ -141,6 +141,7 @@
 #### Next.JS
 
 * [Next.js Material UI Malayalam Tutorial](https://www.youtube.com/playlist?list=PL5Y_OOpi0rh0VAfV9Lz7gTdxhPTkmNxeC) -  Tutorial Hut
+* [Next JS 13 - 2023 Malayalam Tutorial For Beginners](https://www.youtube.com/watch?v=MIIsaNyQRak&list=PLCOzHVsG8mkoY7JQC44AkhsIQnN_f_-aS) -  Web diary
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
```suggestion
* [Next JS For Beginners](https://www.youtube.com/playlist?list=PLCOzHVsG8mkoY7JQC44AkhsIQnN_f_-aS) -  Web diary
```